libraries/AP_HAL_ChibiOS/hwdef/CubeRedPrimary-PPPGW/defaults.parm

```diff
@@ -0,0 +1,14 @@
+NET_ENABLED 1
+NET_OPTIONS 1
+
+# enable hw flow control
+UART1_RTSCTS 1
+
+# swap TX and RX
+UART1_OPTIONS 8
```

Comment:
These are swapped?!?!?! @bugobliterator please confirm

Reply:
this is a PPP gw. With a straight cable we need to swap. Once CubePilot has a final design for the adapter and if that design does the swap in hardware then we can change this